tests/test_main.py

```diff
@@ -12,10 +14,17 @@
         ("Abracadabra", "A", 5),
```

Comment:
The test case for 'Abracadabra' and 'A' should expect a count of 2, not 5. This is an error related to the task requirements.